src/components/MoteresponsColumn.tsx

```diff
@@ -34,6 +40,12 @@ function ResponsTag({ respons }: { respons: DeltakerRespons }): ReactElement {
         Har åpnet
       </Tag>
     );
+  } else if (deltaker === "Behandler" && !respons.harLest && !respons.svar) {
+    return (
+      <Tag variant="alt2" size="xsmall">
+        Ikke svart
+      </Tag>
```

Comment:
Vi vet ikke om legen har åpnet eller ikke, så det "riktige" er å bruke `Ikke svart` i stedet for `Ikke åpnet`